README.md

```diff
@@ -56,10 +30,31 @@ A Gutter Game is when the player never hits a pin (20 zero scores).
 
 ### Perfect Game
 
-A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
-
-In the image below you can find some score examples.
-
-More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
-
-![Ten Pin Score Example](images/example_ten_pin_scoring.png)
+A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.  
+
+## Getting Started
+
+This program runs in the terminal console allowing for user input.
+
+To begin,clone the repository from GitHub:
+```
+$ git clone https://github.com/PiperS52/bowling-challenge-ruby.git
+$ cd bowling-challenge-ruby
+```
+Load irb:
+```
+$ irb -r './lib/scorecard.rb'
+```
+Create a new scorecard:
+```
+scorecard = ScoreCard.new
+scorecard.run
+```
+### Completed
+- Scores are correct for ten frames (inc bonus)
+- Programs runs as intended
+
+### Queries
+- Any guidance on how to implement further feature tests, as requiring user input to generate scores seems to make this slightly more fiddly (?)
```

Comment:
This [answer](https://stackoverflow.com/a/46515819) shows a way of simulating multiple user inputs. Does this help when considering how to make feature tests?